### PR TITLE
HacktoberFest 2019 pullRequest demo

### PR DESCRIPTION
**Pull request by cruzcid**
Haiku pull request POC

### CODE REVIEW

File: _haikus/enligth.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: Data knowledge
+author: cruz
+---
+
+please serve the content<br>
+data will spread knowledge<br>
```
Comment (ltagliaferri) [inline _haikus/enligth.md]: This line needs an extra syllable, maybe something like:

```
this data will spread knowledge <br>
```

File: _haikus/enligth.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+gave birth to light above<br>%
```
Comment (ltagliaferri) [inline _haikus/enligth.md]: This line is one syllable too long and has an extra character, you can try:

```
birthing light above <br>
```

File: _haikus/enligth.md (hunk 1 above)
Comment (cruzcid) [reply _haikus/enligth.md]: I really appreciate your help and assistance. I'll change it ASAP.

File: _haikus/enligth.md (hunk 2 above)
Comment (cruzcid) [reply _haikus/enligth.md]: Once again thank you so much for your time. You are a hero!

Comment (cruzcid) [reply _haikus/enligth.md]: I have already submit the corrections. Hopefully this time will pass the texts :D

**Comment by ltagliaferri** [comment]
Great, thank you for making these changes and for contributing to open source!

**Pull request merged**